.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: 'bug: [description]'
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
```

Comment:
making these html comments makes it less likely that these are still in the submitted issue